lib/jormungandr/src/Cardano/Wallet/Jormungandr/Network.hs

```diff
@@ -75,9 +88,9 @@ data JormungandrLayer m = JormungandrLayer
     { getTipId
         :: ExceptT ErrNetworkUnreachable m BlockId
     , getBlock
-        :: BlockId -> ExceptT ErrGetBlock m Block
+        :: BlockId -> ExceptT ErrNetworkUnreachable m (Maybe Block)
```

Comment:
`Nothing` denotes that the node is not aware of a block with that hash. Maybe put that in a comment.

Reply:
As discussed in another review, returning `Maybe` here isn't very ideal since in most (all?) cases, we are actually expecting a block when calling `getBlock`. Using `Maybe`, we are loosing information (hence the need for an extra comment). With a clear error, there's no need for a comment or any doubt about what the behavior of that function is.